### ISSUE TITLE
Docu of how to use batch effect is misleading

### ISSUE THREAD

**Issue by SusiJo** [qbic-pipelines/rnadeseq#155]
### Description of the bug

The documentation states that one can control for batch effects when adding `batch` to the design, respectively `linear_model.tsv` and to the `samplesheet.tsv`.
Currently, the format in the docu states it should be this format:
`~ batch + condition_variable_of_interest`

And the corresponding `samplesheet.tsv` should have a column called `batch`.

This will trigger the following error because the column `batch` is not added to the `coldata` (see https://github.com/qbic-pipelines/rnadeseq/blob/c75cf3675509397bd4db19a6ce1d6c6f8cebca08/bin/DESeq2.R#L214)

```
Error in DESeqDataSet(se, design = design, ignoreRank) : 
    all variables in design formula must be columns in colData
  Calls: DESeqDataSetFromTximport -> DESeqDataSetFromMatrix -> DESeqDataSet
```

Suggested solution: 
* the `linear_model.tsv` should contain: `~  condition_batch + condition_variable_of_interest`
* the `samplesheet.tsv` batch column should be labelled `Condition: batch`

Another problem is the testing of the test_batcheffect.config with the current release:
```
nextflow run qbic-pipelines/rnadeseq -r 2.0 -profile test_batcheffect,cfc 
--> Unknown configuration profile: 'test_batcheffect'
```

Strangely it works, however, when specifying `-r master` instead.

**Comment by WackerO** [comment]
Note to self: Should also go over the entire docu and see if something needs to be improved/completed

**Comment by WackerO** [comment]
Closed with #159
